main.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exclusive - E-commerce</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <div class="top-bar">Summer Sale For All Swim Suits And Free Express Delivery - OFF 50%! <a href="#">ShopNow</a></div>
+        <nav>
+            <h1 class="logo">Exclusive</h1>
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Sign Up</a></li>
+            </ul>
+            <div class="search-bar">
+                <input type="text" placeholder="What are you looking for?">
+                <i class="fas fa-search"></i>
+                <i class="fas fa-heart"></i>
+                <i class="fas fa-shopping-cart"></i>
+                <i class="fas fa-user"></i>
+            </div>
+        </nav>
+    </header>
+    
+    <section class="banner">
+        <div class="banner-content">
+            <h2>Up to 10% off Voucher</h2>
+            <button class="btn-black">Shop Now →</button>
+        </div>
+    </section>
+    
+    <section class="products">
+        <div class="section-header">
+            <span class="highlight">This Month</span>
+            <h2>Best Selling Products</h2>
+        </div>
+        <div class="product-list">
+            <div class="product">
+                <img src="./img/Frame 605.png" alt="The north coat">
+                <h3>The north coat</h3>
+                <p><span class="price">$260</span> <span class="old-price">$360</span></p>
+                <div class="rating">⭐⭐⭐⭐⭐ (65)</div>
+            </div>
+            <div class="product">
+                <img src="./img/Frame 606.png" alt="Gucci duffle bag">
+                <h3>Gucci duffle bag</h3>
+                <p><span class="price">$960</span> <span class="old-price">$1160</span></p>
+                <div class="rating">⭐⭐⭐⭐⭐ (65)</div>
+            </div>
+            <div class="product">
+                <img src="./img/Frame 610.png" alt="RGB liquid CPU Cooler">
+                <h3>RGB liquid CPU Cooler</h3>
+                <p><span class="price">$160</span> <span class="old-price">$170</span></p>
+                <div class="rating">⭐⭐⭐⭐⭐ (65)</div>
+            </div>
+            <div class="product">
+                <img src="./img/Frame 612.png" alt="Small BookShelf">
+                <h3>Small BookShelf</h3>
+                <p><span class="price">$360</span> <span class="old-price">$400</span></p>
+                <div class="rating">⭐⭐⭐⭐⭐ (65)</div>
+            </div>
+        </div>
+        <button class="btn-red">See more</button>
```

Comment:
Aqui há um problema de espaçamento.

Você pode adicionar espaços entre os itens usando esta técnica: [Axiomatic CSS and Lobotomized Owls](https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/).

Exemplo:

```css
.products * + * {
    margin-bottom: 15px;
}
```